ts/index.ts

```diff
@@ -6,6 +6,7 @@
 */
 
 import { Iterable } from './Iterable'
+import IterableInAppManager from './IterableInAppManager'
```

Comment:
nit pick: inconsistent with above line. {} present in line 8 while its not used in line 9

Reply:
IterableInAppManager is a default export, while Iterable is a named export. see [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export).